Cargo.toml

```diff
@@ -12,12 +12,14 @@ repository    = "https://github.com/Lay3rLabs/avs-toolkit"
 lavs-mock-operators = { path = "contracts/mock-operators" }
 lavs-task-queue = { path = "contracts/task-queue" }
 lavs-verifier-simple = { path = "contracts/verifier-simple" }
+verifier-helper = { path = "tools/verifier-helper"}
```

Comment:
Move to packages

Reply:
fixed in b210cc859bc966f461136a27803b355de9717c89